src/pages/trade-types/multiplier/_what-are-options.tsx

```diff
@@ -385,19 +386,31 @@ const WhatAreOptions = () => {
                 </SmallContainer>
             </StyledSectionContainer>
 
-            {is_uk ? (
+            <UK>
                 <AvailableTrades
                     display_title={localize('Instruments available to trade on Multipliers')}
                     Forex={CFDs}
                 />
-            ) : (
+            </UK>
+
+            <EU>
+                <AvailableTrades
+                    display_title={localize('Instruments available to trade on Multipliers')}
+                    Forex={CFDs}
+                    SyntheticIndices={SyntheticIndices}
+                    Cryptocurrencies={Cryptocurrencies}
+                />
+            </EU>
+
+            <ROW>
                 <AvailableTrades
                     display_title={localize('Instruments available to trade on Multipliers')}
                     Forex={CFDs}
                     SyntheticIndices={SyntheticIndices}
                     Cryptocurrencies={Cryptocurrencies}
+                    BasketIndices={BasketIndices}
                 />
-            )}
+            </ROW>
```

Comment:
Instead of duplication of rendering the components we can simplify it in one component 

```
<AvailableTrades
                    display_title={localize('Instruments available to trade on Multipliers')}
                    Forex={CFDs}
                    SyntheticIndices={is_row || is_eu ? SyntheticIndices : null}
                    Cryptocurrencies={is_row || is_eu ? Cryptocurrencies : null}
                    BasketIndices={is_row ? BasketIndices: null}
                />
```

Reply:
Good idea, thanks!